### ISSUE TITLE
`execute` is quite verbose

### ISSUE THREAD

**Issue by justin-caldicott** [javadparvaresh/dotnet-exec#37]
It would be great to run commands via just `dotnet exec` too to save the fingers.

(Love the concept btw. Coming back to .NET after a while of node, I was looking for something like npm scripts)

**Comment by ivanparvaresh** [comment]
@justin-caldicott yeah, dotnet exec has already reserved by the don't core, unfortunately, it's not possible to take that alias. 
And yeah, the main idea of this tool is to bring `pakcage.json#scripts` concept to dotnet

**Comment by justin-caldicott** [comment]
That's a shame. What about `dotnet do`? Obviously `run` and `script` are taken..

**Comment by ivanparvaresh** [comment]
@justin-caldicott `do` makes sense, Thanks
